src/schemas/json/codecov.json

```diff
@@ -0,0 +1,257 @@
+              "type": "object",
+              "properties": {
+                "url": { "type": "string" },
+                "branches": { "type": "string" },
+                "threshold": { "type": "string" },
+                "message": { "type": "string" },
+                "flags": { "type": "string" },
+                "base": { "enum": ["parent", "pr", "auto"] },
+                "only_pulls": { "type": "boolean" },
+                "paths": { "type": "array", "items": { "type": "string" } },
+                "card": { "type": "boolean" },
+                "notify": { "type": "boolean" }
+              }
+            },
+            "webhook": {
+              "type": "object",
+              "properties": {
+                "url": { "type": "string" },
+                "branches": { "type": "string" },
+                "threshold": { "type": "string" },
+                "message": { "type": "string" },
+                "flags": { "type": "string" },
+                "base": { "enum": ["parent", "pr", "auto"] },
+                "only_pulls": { "type": "boolean" },
+                "paths": { "type": "array", "items": { "type": "string" } }
+              }
+            },
+            "email": {
+              "type": "object",
+              "properties": {
+                "url": { "type": "string" },
+                "branches": { "type": "string" },
+                "threshold": { "type": "string" },
+                "message": { "type": "string" },
+                "flags": { "type": "string" },
+                "base": { "enum": ["parent", "pr", "auto"] },
+                "only_pulls": { "type": "boolean" },
+                "paths": { "type": "array", "items": { "type": "string" } },
+                "layout": { "$ref": "definitions/layout" },
+                "+to": { "type": "array", "items": { "type": "string" } }
+              }
+            }
+          }
+        },
+        "status": {
+          "description": "Commit status. See https://docs.codecov.io/docs/commit-status for details.",
+          "type": ["boolean", "object"],
+          "properties": {
+            "project": {
+              "type": ["boolean", "object"]
+            }
+          },
+          "patch": {
+            "type": ["boolean", "object"]
+          },
+          "changes": {
+            "type": ["boolean", "object"]
+          }
+        }
+      }
+    },
+    "ignore": {
+      "description": "Ignoring paths. see https://docs.codecov.io/docs/ignoring-paths for details.",
+      "type": "array",
+      "items": { "type": "string" }
+    },
+    "fixes": {
+      "description": "Fixing paths. See https://docs.codecov.io/docs/fixing-paths for details.",
+      "type": "array",
+      "items": { "type": "string" }
+    },
+    "flags": {
+      "description": "Flags. See https://docs.codecov.io/docs/flags for details.",
+      "type": "array",
+      "items": { "$ref": "definitions/flag" }
```

Comment:
Feedback welcome. I'm not sure I did this correctly, I need a mapping of flags here